specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1960,6 +2281,124 @@
         "id"
       ],
       "description": "Virtual Network rule."
+    },
+    "Snapshot": {
+      "type": "object",
+      "description": "Response for Volume Snapshot request.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of Volume Snapshot.",
+          "$ref": "#/definitions/SnapshotProperties"
+        },
+        "systemData": {
```

Comment:
>systemData [](http://example.com/codeflow?start=9&length=10)

systemData would be included automatically if you inherit ProxyResource from common types